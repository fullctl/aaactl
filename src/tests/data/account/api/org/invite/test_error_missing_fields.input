{
  "client": "api_client",
  "data": {}
}
