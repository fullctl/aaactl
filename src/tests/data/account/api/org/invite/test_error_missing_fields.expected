{
  "status": 400,
  "response": {"data": [], "errors": {"email": ["This field is required."]}, "profiling": {"queries": 0}}
}
